lib/kql/kql/parser.py

```diff
@@ -376,7 +377,14 @@ def lark_parse(text):
     walker = BaseKqlParser(text)
 
     try:
-        return lark_parser.parse(text)
+        tree = lark_parser.parse(text)
+
+        # Check for whitespace around "and" and "or" tokens
```

Comment:
I know where checking for whitespace, but are there other operators that always require whitespace that needs to be addressed? e.g. `==`